telemetry/specs/INT.mdk

```diff
@@ -384,12 +384,36 @@ ignore the designated bit position.)
 * TCP/UDP destination Port field: A port number in layer 4 will be reserved to
 indicate the existence of INT after TCP/UDP. INT source changes the destination
 Port, and sink must restore the original port number.
+* New Probe Marker fields: arbitrary 64-bit values are inserted after IP TCP/UDP header
+to indicate the existence of INT after TCP/UDP. These fileds should be interpreted
+as unsigned integer values, stored in network byte order and are initialized to a 
+configured value.  This Probe marker is a variation of an early IETF draft
+https://tools.ietf.org/html/draft-lapukhov-dataplane-probe-01 (expired), with existing
+implemetations.
```

Comment:
We can leave out reference to an expired draft. This approach is justified even without support from the draft.

Reply:
It's mainly for pointing it out that there is implementation based on that.